index.html

```diff
@@ -42,5 +43,185 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
       </ul>
     </div>
   </section>
+
+<!-- Section: Recent Works -->
+<section class="works_main">
+
+  <div class="works_title">
+    <h3 class="Title-work-main Title2_dark">My recent Works</h3>
+    <span class="line_div"></span>
+  </div>
+
+  <div class="main_card_work">
+    <div class="main_card">
+      <div class="cnt_img_main_card">
+        <img class="work_img" src="/images/card1_img.png" alt="Project image">
+      </div>
+      <div class="cnt_info_main_card">
+        <h3 class="main_work Title2_dark">Multi-Post Stories</h3>
+        <p class="Simpletext">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="list_feat list_card_1st">
+          <li class="li_btn">CSS</li>
+          <li class="li_btn">HTML</li>
+          <li class="li_btn">BOOTSTRAP</li>
+          <li class="li_btn">RUBY</li>
+        </ul>
+        <a class="project_btn_all project_btn_1" href="#">See Project</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="grid_works">   
```

Comment:
It is also a project so consider adding it to include in the same project section you used with the class of `grid_works` on the container to meet the project requirements 👍

Reply:
Don, thank You @ABDULALI3468